apps/files/src/FileUpload.js

```diff
@@ -31,10 +32,14 @@ class FileUpload {
     xhr.upload.addEventListener('progress', (e) => {
       this.onProgress(e, this.file)
     }, false)
+
     const promise = new Promise((resolve, reject) => {
       xhr.onload = e => {
         xhr.status >= 200 && xhr.status < 400 ? resolve(e) : reject(new Error(xhr.statusText))
       }
+      xhr.onloadend = e => {
```

Comment:
is this polyfilled for IE and co ? see https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onloadend

Reply:
AFAIK using onload should be enough here ?

Reply:
Tested it in ie11 - works.

The good thing about onloadend is that it's triggered also in case it failed.

Reply:
ok, so if it works it means we have a polyfill in place.

note that chrome also doesn't support this property, mind testing there as well ?

Reply:
Already tested. Works in IE11, FF, chrome, safari and edge.